autoreject/autoreject.py

```diff
@@ -668,7 +664,8 @@ def fit(self, epochs):
             if len(self.picks) < 4:
                 raise ValueError('Too few channels. autoreject is unlikely'
                                  ' to be effective')
-            max_interp = min(len(self.picks), 32)
+            # XXX: dont interpolate all channels
+            max_interp = min(len(self.picks) - 1, 32)
```

Comment:
@agramfort this was the problem. Autoreject was sometimes trying to interpolate all the channels. Inside MNE, there is no check for this and it gets stuck